FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -170,6 +172,10 @@ extension MessageViewController {
                 isEmail: true
             )
             let decryptErrBlocks = decrypted.blocks.filter { $0.decryptErr != nil }
+            let decryptAttBlocks = decrypted.blocks.filter { $0.type == .plainAtt || $0.type == .encryptedAtt || $0.type == .decryptedAtt }
+
+            let attachments = decryptAttBlocks.map { Attachment(name: $0.attMeta?.name ?? "Attachment", size: $0.attMeta?.length ?? 0) }
```

Comment:
same for Attachment. You can have
``` 
extension Attachment {
    init(block: MsgBlock) {
    self.name = 
    self.size = 
    }
}
```
and then it can be just
```
let attachments = decrypted.blocks
       .filter(.isAttributeBlock)
       .map(Attachment.init)
```

this solution is more flexible and can be tested easily.